continuous_integration/environment-3.6.yaml

```diff
@@ -31,7 +31,7 @@ dependencies:
   - boto3
   - botocore
   - aiohttp
-  - bokeh
+  - bokeh<2.3
```

Comment:
https://github.com/dask/dask/issues/7292

Reply:
We should be able to remove this version constraint now that https://github.com/dask/dask/pull/7297 is in

Reply:
done